src/app/routes/utils/regex/index.test.js

```diff
@@ -503,7 +503,9 @@ describe('frontPage -> homePage migration', () => {
 
   const serviceToRoute = service => `/${service}`;
 
-  const homepageServices = worldServices.map(serviceToRoute);
+  const homepageServices = worldServices
+    .filter(service => service !== 'zhongwen')
+    .map(serviceToRoute);
```

Comment:
We need to revert this before merging
```suggestion
  const homepageServices = worldServices.map(serviceToRoute);
```